### ISSUE TITLE
Rlsbb 2.0.1 failing to install

### ISSUE THREAD

**Issue by sb2780** [bugatsinho/bugatsinho.github.io#59]
I was not able to update from version 2.0, deleted and tried to do a fresh install from the Repo. I keep getting a "failed to install" message. Checked the log and it says:

Error: Failed to read 'zip://......plugin.video.releasebb-2.0.2.zip/plugin.video.releaseBB/addon.xml'
Error: CAddonInstallJob[plugin.vido.releaseBB]:invalid package special://home/addons/packages/plugin.video.releaseBB-2.0.1.zip

Any suggestions? Other add-ons in the repo installed fine.

**Comment by pitsi** [comment]
I got this too, but when releasebb tries to update to 2.0.1. This is what is mentioned in the log, which is practically the same as above.
```
2020-11-21 08:58:12.323 T:140614662346496   ERROR: Failed to read 'zip://special%3a%2f%2fhome%2faddons%2fpackages%2fplugin.video.releaseBB-2.0.1.zip/plugin.video.releaseBB/addon.xml'
2020-11-21 08:58:12.323 T:140614662346496   ERROR: CAddonInstallJob[plugin.video.releaseBB]: invalid package special://home/addons/packages/plugin.video.releaseBB-2.0.1.zip
```
I think the entire zip of 2.0.1 is faulty, because it lacks a ton of things, not just addon.xml that is mentioned above.

Contents of plugin.video.releaseBB-2.0.1.zip
![2020-11-21-090328_538x427_scrot](https://user-images.githubusercontent.com/5880037/99870020-71f8e780-2bd8-11eb-939b-1638126f7b75.png)

Contents of plugin.video.releaseBB-2.0.0.zip
![2020-11-21-090423_538x427_scrot](https://user-images.githubusercontent.com/5880037/99870034-9228a680-2bd8-11eb-9eb7-88414c5fa587.png)

**Comment by pitsi** [comment]
Fixed in v 2.0.1.**1**, so this can be closed now.
Contents of 2.0.1.1

![2020-11-21-162719_538x427_scrot](https://user-images.githubusercontent.com/5880037/99879577-907dd380-2c16-11eb-9da1-761ea185b703.png)
